docs/pages/1_user/02_connect_me.md

```diff
@@ -74,10 +76,14 @@ The following steps describe how to download and install the Shieldoo desktop cl
 
 
 ## Connect to Shieldoo Secure Network
-This section describes establishing a secure connection to a server and attached services in Shieldoo Secure Network.
+This section describes establishing a secure connection to a server and attached services in Shieldoo Secure Network:
+1. [Connect to Servers](/connect_me/#connect-to-servers)
+2. [Connect to the Attached Services](/connect_me/#connect-to-the-attached-services)
 
-- __Servers__ are devices to which users or other servers can connect.
-- __Attached Services__ are a device that runs on the local network of a server but on which Shieldoo Secure Network cannot be installed (e.g. a printer). Shieldoo lets you directly connect to such services without connecting to their servers. These services are managed by the admin of Shieldoo Secure Network.
+{: .note }
+>__Servers__ are devices to which users or other servers can connect.
+>
+>__Attached Services__ are a device that runs on the local network of a server but on which Shieldoo Secure Network cannot be installed (e.g. a printer). Shieldoo lets you directly connect to such services without connecting to their servers. These services are managed by the admin of Shieldoo Secure Network.
```

Comment:
Tady mas "Attached Services are" - cili mnozne cislo, ale pak "a device that runs..." cili jednotne. Chtelo by to sladit.

Reply:
Máš pravdu. Opravím to.